lib/internal/worker.js

```diff
@@ -212,13 +211,15 @@ class Worker extends EventEmitter {
 
     debug('instantiating Worker.', `url: ${url}`, `doEval: ${doEval}`);
     // Set up the C++ handle for the worker, as well as some internal wiring.
-    this[kHandle] = new WorkerImpl(url,
-                                   env === process.env ? null : env,
-                                   options.execArgv,
-                                   parseResourceLimits(options.resourceLimits),
-                                   !!(options.trackUnmanagedFds ?? true),
-                                   isInternal,
-                                   name);
+    this[kHandle] = new WorkerImpl(
+      url,
+      env === process.env ? null : env,
+      options.execArgv,
+      parseResourceLimits(options.resourceLimits),
+      !!(options.trackUnmanagedFds ?? true),
+      isInternal,
+      name,
+    );
```

Comment:
Can you please revert that to avoid polluting `git blame`?

Reply:
> isn't a convincing reason to leave it how it was

What's a reason not to leave it as it was? If there's no reason to make a change and at least one good reason not to do it, it kinda doesn't matter if you find it convincing, let's not do it

Reply:
The continuing indentation made the lines hard-to-read, and doesn't follow the convention set by the layout of e.g. `module.exports`

The rule I follow (and nodejs codebase largely seems to as well) - if each parameter is on its own line, a newline should precede the first parameter (and follow the last one, after a comma)

Reply:
This kind of cosmetic changes should come with a linter rule, which would be a different PR. Please revert this change.